app/containers/MessageBox/index.js

```diff
@@ -109,12 +104,11 @@ class MessageBox extends Component {
 		this.customEmojis = [];
 		this.onEmojiSelected = this.onEmojiSelected.bind(this);
 		this.text = '';
-		this.fileOptions = [
-			I18n.t('Cancel'),
-			I18n.t('Take_a_photo'),
-			I18n.t('Take_a_video'),
-			I18n.t('Choose_from_library'),
-			I18n.t('Choose_file')
+		this.options = [
```

Comment:
We have to change these icons and I'm talking to design team about it.
We don't have these icons yet.